Description: Fixed segfault when renaming or removing folder in project.
  When adding folder from filesystem to project and creating new folder in this folder, removing or renaming causes segfault.
  This happens because new folder must have flag 'is_grafted' (project node not from real file or folder), but code in brasero-data-project.c in function brasero_data_project_add_node_real doesn't handle nodes with flag 'is_fake' correctly (new empty folder has this flag). Node isn't marked as grafted with function brasero_file_node_graft. I've just added call of this function.
Author: Andrej Belym <white.wolf.2105@gmail.com>
Index: brasero-3.2.0/libbrasero-burn/brasero-data-project.c
=============================================================
--- brasero-3.2.0.orig/libbrasero-burn/brasero-data-project.c
+++ brasero-3.2.0/libbrasero-burn/brasero-data-project.c
@@ -1674,6 +1674,7 @@ brasero_data_project_add_node_real (Bras
 		/* The node is a fake directory; graft it as well as all the 
 		 * nodes already in the tree with the same URI */
 		graft = brasero_data_project_uri_graft_nodes (self, uri);
+		brasero_file_node_graft (node, graft);
 	}
 	else {
 		gchar *parent_uri;
